tracing-subscriber/src/fmt/env_logger_compat.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{EnvFilter, FmtSubscriber};
+use std::{error::Error, fmt};
+use tracing::subscriber::{self, SetGlobalDefaultError};
+
+#[cfg(feature = "tracing_log")]
+use tracing_log::{LogTracer, SetLoggerError};
+
+#[derive(Debug)]
+enum InitError {
```

Comment:
TIOLI: this could probably just be a `Box`; this code won't get called in the hot path. I like the idea of returning `impl Error` to hide the private type, though...